src/main/scala/Garage/MongoConnection.scala

```diff
@@ -0,0 +1,35 @@
+package Garage
+
+import org.mongodb.scala._
+import org.mongodb.scala.model.Filters._
+import org.mongodb.scala.model.Updates._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{Failure, Success}
+
+class MongoConnection {
+  val mongoClient: MongoClient = MongoClient("mongodb://localhost")
```

Comment:
don't think it's a good idea to hard code the client and database, it would be better to have separate methods to get them based on the input

Reply:
I was going to move them to a constants file, do you want me to have

getMongoClient(Int)
Pattern match 
      case 0
     case 1

Something like that